src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -283,7 +283,7 @@ protected function instantiateObject(array $data, $class, array &$context, \Refl
                 } elseif ($constructorParameter->isOptional()) {
                     $params[] = $constructorParameter->getDefaultValue();
                 } else {
-                    throw new RuntimeException(
+                    throw new \RuntimeException(
```

Comment:
I'll do another PR for the tests to not loose the "vacuum cleaner", if you don't mind :)